app/models/concerns/locatable.rb

```diff
@@ -0,0 +1,11 @@
+module Locatable
+  extend ActiveSupport::Concern
```

Comment:
[Concerns](http://api.rubyonrails.org/classes/ActiveSupport/Concern.html) are an interesting feature for abstracting common functionality (described ages ago [in this blog post](https://signalvnoise.com/posts/3372-put-chubby-models-on-a-diet-with-concerns)).

I've tried to use them sparingly, but I think this is a good use case for them.

Reply:
ah it's like traits in php